### PR TITLE
fix/flaky tests

### PR DESCRIPTION
**Pull request by Magnus-Kuhn**
extend synchronisation time

### CODE REVIEW

File: packages/runtime/test/lib/testUtils.ts [hunk 1]
```diff
@@ -61,9 +62,9 @@ export async function syncUntil(transportServices: TransportServices, until: (sy
 
         iterationNumber++;
         criteriaMet = until(finalSyncResult);
-    } while (!criteriaMet && iterationNumber <= 10);
+    } while (!criteriaMet && iterationNumber <= timeoutMs / iterationSleepMs);
 
-    if (!criteriaMet) throw new Error("syncUntil: the criteria specified in syncUntil were not fulfilled");
+    if (!criteriaMet) throw new Error(`syncUntil: the criteria specified in syncUntil were not fulfilled, iterations run: ${iterationNumber}`);
```
Comment (slavistan) [inline packages/runtime/test/lib/testUtils.ts:67]: The iterations are quite meaningless. Use the timeout instead: `` `syncUntil timed out after ${timeoutMs} ms.` ``.

Comment (slavistan) [reply packages/runtime/test/lib/testUtils.ts:67]: Does this fix all flaky tests or are we "just" making progress? :-)

Comment (Magnus-Kuhn) [reply packages/runtime/test/lib/testUtils.ts:67]: this has run successfully twice in all variants on my computer - so I'd say it's done

File: packages/runtime/test/lib/testUtils.ts
Comment (jkoenig134) [inline packages/runtime/test/lib/testUtils.ts:1]: Wouldn't the same fix to increase the iterations to 20 or more? This is quite unreadable as you now compare iterationNumber to the timeout

Comment (jkoenig134) [reply packages/runtime/test/lib/testUtils.ts:1]: in this case iteration number would be **160**, quite high.

File: packages/runtime/test/lib/testUtils.ts (hunk 1 above)
Comment (jkoenig134) [reply packages/runtime/test/lib/testUtils.ts]: > on my computer 

but the pipeline failed :/

**Review by jkoenig134 — APPROVED** [review]

**Pull request merged**